<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationPage</class>
 <widget class="QWidget" name="RegistrationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Signup</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_signup">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>381</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_signup_username"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_signup_email"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_signup_password"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_signup_confirm"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Login</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_login_username"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_login_password"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
